src/sentry/static/sentry/app/routes.jsx

```diff
@@ -750,6 +748,17 @@ function routes() {
               import(/*webpackChunkName:"OrganizationDiscover"*/ './views/organizationDiscover/index')}
             component={errorHandler(LazyLoad)}
           />
+          <Redirect
```

Comment:
Never unless you navigate directly to it in your browser. But I thought it's nicer for this to be a valid path since we're adding the `/saved/:id` path for the child resources